### PR TITLE
Doc save model

### PR DESCRIPTION
**Pull request by SunMarc**
# What does this PR do ? 
This PR modifies the saving/loading section of Quicktour documentation. It replaces the `save` method by the `save_model` method from `Accelerator` class.

### CODE REVIEW

**Review by muellerzr — APPROVED** [review]
Thanks for the update on the docs!

File: docs/source/quicktour.md [hunk 1]
```diff
@@ -348,27 +348,39 @@ point in the script as shown above, and then, you should unwrap your model befor
...
 
 If your script contains logic to load a checkpoint, we also recommend you load your weights in the unwrapped model
 (this is only useful if you use the load function after making your model go through
 [`~Accelerator.prepare`]). Here is an example:
 
-```
+```python
 unwrapped_model = accelerator.unwrap_model(model)
-unwrapped_model.load_state_dict(torch.load(filename))
+unwrapped_model.load_state_dict(torch.load(save_directory+"/pytorch_model.bin"))
```
Comment (sgugger) [inline docs/source/quicktour.md:373]: Here you should use `os.path.join` as this code won't work on Windows.

**Review by sgugger — APPROVED** [review]
Thanks!

File: docs/source/quicktour.md (hunk 1 above)
Comment (SunMarc) [reply docs/source/quicktour.md:373]: great catch =)

File: docs/source/package_reference/utilities.md [hunk 2]
```diff
@@ -105,3 +105,9 @@ These utilities relate to setting and synchronizing of all the random states.
 These include utilities that are useful while using PyTorch with XLA.
 
 [[autodoc]] utils.install_xla
+
+## Load model
```
Comment (muellerzr) [inline docs/source/package_reference/utilities.md:109]:
```suggestion
## Loading model weights
```

**Review by muellerzr — COMMENTED** [review]
Just a mild idea/suggestion on the title for the section :) (If you like it though, feel free to ignore!)

File: docs/source/package_reference/utilities.md (hunk 2 above)
Comment (SunMarc) [reply docs/source/package_reference/utilities.md]: Added! Thx for the suggestion =)

**Pull request merged**